site/ja/tutorials/customization/custom_training.ipynb

```diff
@@ -0,0 +1,444 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "5rmpybwysXGV"
+   },
+   "source": [
+    "##### Copyright 2019 The TensorFlow Authors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "colab": {},
+    "colab_type": "code",
+    "id": "m8y3rGtQsYP2"
+   },
+   "outputs": [],
+   "source": [
+    "#@title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
```

Comment:
[Strong suggestion] It is necessary to replace this with the Japanese version.

Reply:
Do you mean the license term?
I think the open source software license should be the original english version.

Reply:
Sorry, I made a mistake. I thought that it was note about statement of the community translation.